### PR TITLE
Refactor CA Cert secret names

### PR DESCRIPTION
**Pull request by AurelienGasser**
See main PR: https://github.com/SubstraFoundation/hlf-k8s/pull/16

### CODE REVIEW

File: charts/substra-backend/values.yaml [hunk 1]
```diff
@@ -86,7 +86,7 @@ secrets:
   # This should include the Orderer TLS 'cacert.pem'
   orderer:
     tls:
-      ca: hlf-client-tlsrootcert
+      caCert: hlf-cacert
```
Comment (AurelienGasser) [inline charts/substra-backend/values.yaml]: Note that the value we put here doesn't seem to matter.

I set it to:

```
caCert: hlf-idkey
```

and the populate happily succeeded :man_shrugging: 

My understanding is that the backend needs to communicate securely with the orderer in order to commit transactions. But somehow, even with a bad root CA file, the backend-orderer connection succeeds. Maybe something is misconfigured? Or am I missing something?

This is probably out of the scope of this PR, but I thought I'd report it :)

Comment (GuillaumeCisco) [reply charts/substra-backend/values.yaml]: The backend will use a peer which will need to connect to the orderer for `invoke` transaction.
This connection need a CA cert.
Maybe in the backend code, the peer uses a CA cert referenced from elsewhere

Comment (Kelvin-M) [reply charts/substra-backend/values.yaml]: I think we get it from the discovery : https://github.com/SubstraFoundation/substra-backend/blob/dev/backend/backend/settings/deps/ledger.py#L111:#L171

We can remove this as we only stuff to connect to our peer !

Comment (Kelvin-M) [reply charts/substra-backend/values.yaml]: I'm doing a PR for that :)

**Comment by Kelvin-M** [comment]
@AurelienGasser  and @GuillaumeCisco  Do you agree on this version. It's not a refacto but more a renaming !

**Pull request merged**